enhancer.js

```diff
@@ -0,0 +1,137 @@
+module.exports = {
+  success,
+  fail,
+  repair
+};
+
+function typeCheck(item) {
+  if (item.type !== "weapon" && item.type !== "armor") {
+    throw new Error(`Incorrect Item Type of ${item.type}`);
+  }
+}
+
+function enhancementCheck(item) {
+  let next = item.enhancement;
+  next += 1;
+  console.log(next);
+  if (next > 20) {
+    throw new Error("Item Enhancement is at a maximum");
+  } else {
+    if (
+      (item.durability < 25 && item.enhancement <= 14) ||
+      (item.durability < 10 && item.enhancement > 14 && item.enhancement <= 20)
+    ) {
+      throw new Error("Item Durability is too Low for Enhancement");
+    } else {
+      item.enhancement += 1;
+      let name = item.originalName;
+      switch (next) {
+        case 1:
+        case 2:
+        case 3:
+        case 4:
+        case 5:
+        case 6:
+        case 7:
+        case 8:
+        case 9:
+        case 10:
+        case 11:
+        case 12:
+        case 13:
+        case 14:
```

Comment:
Rather than having these incrementing cases, you should use a `default` at the very end. Currently, any values 1-14 do fall on that 15 case, but they could all be handled in a default case instead.